packages/relay/src/lib/eth.ts

```diff
@@ -203,20 +204,16 @@ export class EthImpl implements Eth {
 
   /**
    * Estimates the amount of gas to execute a call.
-   *
-   * TODO: API signature is not right, some kind of call info needs to be passed through:
-   *   "params": [{
-   *     "from": "0xb60e8dd61c5d32be8058bb8eb970870f07233155",
-   *     "to": "0xd46e8dd67c5d32be8058bb8eb970870f07244567",
-   *     "gas": "0x76c0",
-   *     "gasPrice": "0x9184e72a000",
-   *     "value": "0x9184e72a",
-   *     "data": "0xd46e8dd67c5d32be8d46e8dd67c5d32be8058bb8eb970870f072445675058bb8eb970870f072445675"}],
-   */
-  async estimateGas() {
-    // FIXME: For now, we are going to have a rough estimate but in the future we can do something more sophisticated.
+   */
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  async estimateGas(transaction: any, blockParam: string | null) {
```

Comment:
`blockParam` is currently unused hence the other code smell.
If you update the log below to note the params this should fix it

Reply:
The param is there because an external API says we may pass it in.  Removing it or un-naming it as `_` will remove the clarity for what the parameter represents.  Automated analysis lacks that context.